_commands/basics/seq.md

```diff
@@ -3,22 +3,64 @@
 
 seq
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`seq` is used to produce a sequence of numbers
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ seq 3
+1
+2
+3
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `seq [FIRST] [LAST]`
```

Comment:
Looks good, but what about options? Maybe include a small part with useful flags for the command as well.